limiter/garden.yml

```diff
@@ -0,0 +1,54 @@
+# Production container to be used without sync mode.
+kind: Build
+type: container
+name: limiter-prod
```

Comment:
Before we merge this, we should add "limiter-sync" and "limiter-prod" to the ECR repository default list in https://github.com/garden-io/garden-aws-quickstart